### DISCUSSION TITLE
SaaS for eCommerce built with Supabase, NextJs, Twilio & Stripe.

### DISCUSSION THREAD

**Discussion by Probabilista** [supabase/supabase#796] [Show and tell]
I'll build a complete open source SaaS for eCommerce : 

- Shops will be able to offer recurring orders via text messages ( with Twilio )
- Customers will be able to order with one text message like "Coffee for one month"
- Payment will be created on Stripe with a stored card number 
- Order will be automatically pushed to shop ( Shopify, WooCommerce, SquareSpace, BigCommerce supported )
- Shop sends the package.

Authentication and DB provided by Supabase, and a NextJs front-end. Text messages supported by Twilio, and payments by Stripe.

**Comment by kiwicopple** [comment]
Sounds amazing @Probabilista ❤️

**Comment by milestones95** [comment]
Any update on this project? Curious how it went with integrating stripe
